src/main/java/com/exadel/discount/entity/City.java

```diff
@@ -0,0 +1,40 @@
+package com.exadel.discount.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@EqualsAndHashCode(exclude = {"users", "vendorLocations"})
+@ToString(exclude = {"users", "vendorLocations"})
+@Table(name = "cities")
+public class City {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    @Column(name = "id")
+    private UUID id;
+
+    @Column(name = "name", length = 50, nullable = false)
+    private String name;
+
```

Comment:
Add, please, row
@Fetch(FetchMode.JOIN)
for fetching Users by country name with OSIV = false through Query in UserRepository.

Reply:
Done

Reply:
is here 
@fetch(FetchMode.JOIN)
? Do not see yet(

Reply:
Abouve country